### PR TITLE
#164243636 Implement pep8 standards on codebase

### PR DESCRIPTION
**Pull request by ElMonstro**
#### What does this PR do?
Adds testing of code for PEP8 style guide compliance
#### Description of Task to be completed?
- Add PEP8 checking packages
- Add PEP8 testing script to travis.yml
#### How should this be manually tested?
* Clone this repo `git clone -b origin/ch-implement-pep8-164243636 https://github.com/andela/ah-legion-backend.git`
* Install dependencies with the command `pip install -r requirements.txt` from the root folder.
* Run `find ./authors/ -name \*.py -exec flake8 -v {} +`
- VIew travis [build](https://travis-ci.com/andela/ah-legion-backend/builds/102683228)
#### Any background context you want to provide?
PEP8 styled code is highly readable and easier to maintain
#### What are the relevant pivotal tracker stories?
[#164243636](https://www.pivotaltracker.com/story/show/164243636)

### CODE REVIEW

**Review by Etomovich — APPROVED** [review]
LGTM

**Review by RachelleMaina — APPROVED** [review]

**Review by cob04 — APPROVED** [review]
0 violations and 0 reported. Looks good

**Review by MandelaK — APPROVED** [review]
Everything looks okay

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore = W3921
+max-line-length = 80
+exclude = */tests/* */settings/*
```
Comment (abulojoshua1) [inline .flake8]: Add end of file in this doc

File: .travis.yml [hunk 2]
```diff
@@ -23,6 +23,7 @@ script:
 after_script:
   - coverage xml
   - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
+  - find ./authors/ -name \*.py -exec flake8 -v {} +
```
Comment (abulojoshua1) [inline .travis.yml]: Run this command in the before_script.

**Review by abulojoshua1 — APPROVED** [review]
LGTM

**Pull request merged**